src/barril/units/_tests/test_unit_database.py

```diff
@@ -214,20 +215,24 @@ def testCategory(unit_database_custom_conversion):
     quantity = ObtainQuantity("m", "my length")
     formatted_value = FormatFloat("%g", -6e-010)
     with pytest.raises(
-        ValueError,
+        QuantityValidationError,
         match="Invalid value for My Length: %s. Must be > %s."
         % (formatted_value, formatted_value),
-    ):
+    ) as exc_info:
         quantity.CheckValue(-6e-10)
+    e = exc_info.value
+    assert hasattr(e, 'caption') is True
```

Comment:
Please check all the actual values here

Reply:
Sorry, i was in doubt about it, should have asked.

Reply:
And please use double quotes (lint)